src/WeihanLi.Common/WeihanLi.Common.csproj

```diff
@@ -23,6 +24,9 @@
     <PackageReference Include="System.ComponentModel.Annotations" />
     <PackageReference Include="Microsoft.CSharp" PrivateAssets="All" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'net7.0' OR '$(TargetFramework)' == 'net8.0'">
```

Comment:
Maybe we could use `$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))` to simplify this

Reply:
The issue is that we'd want to only reference it for anything before net 9.0, but would not need to be included for net9.0, net10.0, net11.0 etc.

It will still work with the condition excluded, but it would be a dependency for nothing.